packages/phone-number-privacy/combiner/src/signing/get-threshold-signature.ts

```diff
@@ -225,7 +231,7 @@ function requestSignature(
 
 function getMajorityErrorCode(errorCodes: Map<number, number>) {
   if (errorCodes.size > 1) {
-    logger.error(ErrorMessage.INCONSISTENT_SIGNER_RESPONSES)
+    logger.error({ errorCodes }, ErrorMessage.INCONSISTENT_SIGNER_RESPONSES)
```

Comment:
Based on [Bunyan docs](https://github.com/trentm/node-bunyan#log-method-api), it sounds like this should be a named parameter.
```suggestion
    logger.error({ errorCodes: errorCodes }, ErrorMessage.INCONSISTENT_SIGNER_RESPONSES)
```

Reply:
I *think* that the field will assume the name of the variable based on how this is done in the attestation-service https://github.com/celo-org/celo-monorepo/blob/e1a6a102dd1654a6774d47198d29e8ccb2360887/packages/attestation-service/src/db.ts#L105 I'll make sure to check.

Reply:
I've verified that the variable name becomes the name of the parameter